config/vaccines.yml

```diff
@@ -101,7 +101,6 @@ gardasil_9:
 menquadfi:
   brand: MenQuadfi
   type: menacwy
-  discontinued: true
```

Comment:
I think Nimenrix is supposed to be the discontinued one, I think I made a mistake in this commit: https://github.com/nhsuk/manage-vaccinations-in-schools/commit/8651dc9d61dc72d14f57f8717f7377cdd35c6309